### ISSUE TITLE
Web Component Link: Develop Alpha 1

### ISSUE THREAD

**Issue by heymatthenry** [uswds/uswds-next#9]
One of the first components we'll recreate as a Web Component is the `usa-link` (uswds/uswds#5706).

## Necessary Tasks

These are the features that should constitute MVP for the pre-release version, since they will help us get our arms around the architectural choices we'll have to make in developing further components.

- [ ] Add more `vitest` unit tests for component
- [ ] Evaluate component's accessibility shortcomings and document them
- [ ] Add a Storybook story for the component (Depends on #6)

## Stretch Goals

All of these will need to happen at some point, but don't necessarily have to happen for the pre-release version

- [ ] The current WIP implementation (#8) for this component renders a link to the shadow DOM. Consider revising that implementation so that the the full link goes in the component `<slot>`.
- [ ] Figure out how to package the component by itself for distribution (Depends on #5)
- [ ] Remediate any accessibility bugs

**Comment by brunerae** [comment]
This should have been closed by https://github.com/uswds/web-components/pull/8
